CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This is the Developer Changelog for Matomo platform developers. All changes in o
 
 The Product Changelog at **[matomo.org/changelog](https://matomo.org/changelog)** lets you see more details about any Matomo release, such as the list of new guides and FAQs, security fixes, and links to all closed issues. 
 
+## Matomo 3.6.0
+
+### Deprecations
+
+* The API method `Proxy.redirect` has been deprecated. Use rel="nofollow" instead.
```

Comment:
FYI: It is not an API method, but a regular controller action. I wonder if we could rather say like 

> Creating links through the Proxy has been deprecated. Use rel="nofollow" instead.

(or similar)